ocs_ci/ocs/cluster.py

```diff
@@ -1551,6 +1552,25 @@ def is_flexible_scaling_enabled():
     return failure_domain == "host" and flexible_scaling
 
 
+def check_ceph_health_after_add_capacity():
```

Comment:
this function check_ceph_health_after_add_capacity should accept param for future use

Reply:
Fixed